src/ngraph_encapsulate_op.cc

```diff
@@ -212,6 +213,29 @@ class NGraphEncapsulateOp : public OpKernel {
     return Status::OK();
   }
 
+  void mem_usage(double& vm_usage, double& resident_set) {
```

Comment:
Move this to some utility file such as ngraph_utils.h. Also follow the proper naming convention.

Reply:
changed to `MemoryProfile` and moved to ngraph_utils.h and ngraph_utils.cc